resources/local/chainspec.toml

```diff
@@ -24,6 +24,12 @@ auction_installer_path = '../../target/wasm32-unknown-unknown/release/auction_in
 accounts_path = 'accounts.csv'
 # Number of slots available in validator auction.
 validator_slots = 5
+# Auction delay. 
+auction_delay = 3
+# initial era
+initial_era_id = 0
+# Locked funds period
+locked_funds_period = 15
```

Comment:
Could you extend those comments a bit? I'm still confused about these constants:
* Does `locked_funds_period` affect only the genesis validators, or also new added bids?
* Why do we need to have an initial era ID other than 0?
* If I make a high bid in era `e` (before `e`'s switch block), will I be a validator in era `e + auction_delay` or `e + auction_delay + 1`? If I request unbonding in era `e`, I guess I the same applies to when I _won't_ be a validator anymore?
* If I request unbonding in era `e`, will I receive my funds in the switch block of era `e + locked_funds_period`? Or `e + locked_funds_period + 1`?

Also, it's probably best to have the exact same comments in `local` and `charlie`.

Reply:
I updated the comments for `locked_funds_period`, as for the effects, I am unclear, I just tried to move them out into the chainspec without breaking anything else

Reply:
Maybe @mpapierski can clarify, or add documentation?

Reply:
@mpapierski: ping

Reply:
> Could you extend those comments a bit? I'm still confused about these constants:
> 
>     * Does `locked_funds_period` affect only the genesis validators, or also new added bids?
> 

Newly added bids through `add_bid` are unlocked by default. Locked period is for validators: https://github.com/CasperLabs/casper-node/blob/d372f5b9bae91a9362e9b240ca6cb98060557ba5/types/src/auction.rs#L105-L113

>     * Why do we need to have an initial era ID other than 0?

We don't - I'm fine keeping it as a constant

> 
>     * If I make a high bid in era `e` (before `e`'s switch block), will I be a validator in era `e + auction_delay` or `e + auction_delay + 1`? If I request unbonding in era `e`, I guess I the same applies to when I _won't_ be a validator anymore?

You will be validator in `e + auction_delay + 1` same for unbonding (aka withdraw_bid). 

> 
>     * If I request unbonding in era `e`, will I receive my funds in the switch block of era `e + locked_funds_period`? Or `e + locked_funds_period + 1`?
> 
> 
> Also, it's probably best to have the exact same comments in `local` and `charlie`.

Your unbonded money will be delivered to you in `e + DEFAULT_UNBONDING_DELAY` era but as a genesis validator you can't withdraw your bid before `e + locked_funds_period`

Reply:
As per changes I've made based on feedback `initial_era_id` is a constant *again*

Reply:
cc @afck: please have a look at @mpapierski's response

Reply:
Yes, thanks for the clarifications!
Let's put these in the chainspecs as comments, and ideally also on the corresponding fields in the `Chainspec` struct.

Reply:
I'm with Andreas here. Consensus and contracts have to be in-sync on this one. Otherwise, we end up with a security hole where validators can unbond while still participating in consensus. I prefer to err on the safe side here and keep them hardcoded.

Reply:
"If, for some reason" 

The only reason EE would start using upgraded chainspec values would be because the commit_upgrade method were called at a specified block height. Which isn't even wired up currently.

Reply:
I'm with @EdHastingsCasperLabs here.  At this point it's probably more likely that someone would tweak a constant without realizing the consequences than it is that we would accidentally launch an upgrade that would change this value.

Reply:
(This is probably about the other thread below: Upgradeability.)

Reply:
Just so we don't forget: This thread was originally about @mpapierski's clarifications:
> Let's put these in the chainspecs as comments, and ideally also on the corresponding fields in the Chainspec struct.

Reply:
Updated comments